.github/workflows/callable-flex-update.yml

```diff
@@ -47,5 +47,9 @@ jobs:
                     git rm -q *.json
                     mv .github/flex-endpoint/*.json .
                     git add *.json
```

Comment:
just after:
```
cp -a .github/flex-endpoint/archived .
git add archived/
```
and remove the rest?

Reply:
Yup, you nailed it - that worked. Thank you for pointing that out :)